exercises/tournament/tournament_test.rb

```diff
@@ -5,65 +5,64 @@
 
 # Common test data version: cda8f98
 class TournamentTest < Minitest::Test
-
   def test_typical_input
     # skip
     input = <<-INPUT.gsub(/^ */, '')
-      Allegoric Alaskans;Blithering Badgers;win
-      Devastating Donkeys;Courageous Californians;draw
-      Devastating Donkeys;Allegoric Alaskans;win
-      Courageous Californians;Blithering Badgers;loss
-      Blithering Badgers;Devastating Donkeys;loss
-      Allegoric Alaskans;Courageous Californians;win
-    INPUT
+Allegoric Alaskans;Blithering Badgers;win
```

Comment:
This doesn't apply in our case since we run CI on ruby `2.1.2` and only require ruby `2.1+`, but I just learned about a cool `2.3+` ruby feature this week!

If you declare your heredoc with `<<~` (`~` rather than `-`), it does this:

```
The indentation of the least-indented line will be removed from each line of the content.
```

Sorry if you'd already seen this, I just got really excited because heredoc indentation has always tripped me up!